test/libsolidity/syntaxTests/inlineAssembly/tstore_warning_only_once_multiple_contracts.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
```

Comment:
I mean, the point of this test was to have more than one contract here to verify the limiting is per compilation, not per contract :) Right now there's only one.

Reply:
Yes, my mistake. I forgot to update the changes before committing. I already have the test with multiple contracts.